api/tests/unit/domain/models/CertificationIssueReportResolutionStrategies_test.js

```diff
@@ -822,6 +823,246 @@ describe('Unit | Domain | Models | CertificationIssueReportResolutionStrategies'
     });
   });
 
+  context('#NEUTRALIZE_IF_TIMED_CHALLENGE', function() {
+
+    context('When challenge is neutralizable', function() {
+
+      it('neutralizes successfully', async function() {
+        // given
+        const certificationChallenge = domainBuilder.buildCertificationChallengeWithType({});
+        const certificationAnswer = domainBuilder.buildAnswer.ko(({ challengeId: certificationChallenge.challengeId }));
+        const certificationAssessment = domainBuilder.buildCertificationAssessment({
+          certificationChallenges: [certificationChallenge],
+          certificationAnswersByDate: [certificationAnswer],
+        });
+        const certificationIssueReport = domainBuilder.buildCertificationIssueReport({
+          subcategory: CertificationIssueReportSubcategories.EXTRA_TIME_EXCEEDED,
+          category: CertificationIssueReportCategories.IN_CHALLENGE,
+          questionNumber: 1,
+        });
+        const timedChallenge = domainBuilder.buildChallenge({
+          timer: 1.33,
+        });
+        const certificationIssueReportRepository = {
+          save: sinon.stub(),
+        };
+        const challengeRepository = {
+          get: sinon.stub().resolves(timedChallenge),
+        };
+
+        // when
+        const neutralizationAttempt = await neutralizeIfTimedChallengeStrategy({ certificationIssueReport, certificationAssessment, certificationIssueReportRepository, challengeRepository });
+
+        // then
+        expect(neutralizationAttempt).to.deep.equal(CertificationIssueReportResolutionAttempt.resolvedWithEffect());
```

Comment:
Pour info, y'a de nouveaux helpers custom chai qui ont enrichi les existants, dont un qui peut me sembler pertinent ici :
il existe maintenant `expect(neutralizationAttempt).to.deepEqualInstance(CertificationIssueReportResolutionAttempt.resolvedWithEffect())`, ça revient à faire la combinaison suivante :
```
expect(neutralizationAttempt).to.be.instanceOf(CertificationIssueReportResolutionAttempt);
expect(neutralizationAttempt).to.deep.equal(CertificationIssueReportResolutionAttempt.resolvedWithEffect());
```